spec/money_spec.rb

```diff
@@ -3,24 +3,29 @@
 
 RSpec.describe "Money" do
 
-  before(:each) do
-    @money = Money.new
-  end
-
+  let (:money) {Money.new(1)}
```

Comment:
👍  for removing this minitest-ism but by changing the value the test description "defaults to 0 when constructed with no arguments" doesn't match anymore.

Reply:
good catch, missed that one, fix incoming

Reply:
done